docs/release-notes.md

```diff
@@ -21,6 +21,7 @@ Minor changes:
 - Add release notes to documentation
 - iso: Detect incomplete ISO files
 - Warn if console kargs could have used `--console`/`--dest-console` instead
+- save-partindex: bail when listed partindex's are located on a MBR disk  
```

Comment:
The prefix should be the subcommand, `install`.  And this implies that we continue if the MBR doesn't contain any of the listed partindexes, which isn't true.  If any partition saving is requested, and the disk is MBR, we bail.

Reply:
Ah, ok. thank oyu